src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,167 @@
     type: command
     short-summary: Creates a remote connection using a tcp tunnel to your web app
 """
+
+helps['functionapp hybrid-connection'] = """
+    type: group
+    short-summary: commands that list, add, or remove hybrid connections from a sepcific functionapp
+"""
+
+helps['functionapp hybrid-connection list'] = """
+    type: command
+    short-summary: lists all the hybrid connections set on the specified functionapp
+    examples:
+        - name: list the hybrid connections
+          text: >
+              az functionapp hybrid-connection list -g [myResourceGroup] -n [myAppName] -s [slotName]
+"""
+
+helps['functionapp hybrid-connection add'] = """
+    type: command
+    short-summary: adds an existing hybrid connection to the specified functionapp
+    examples:
+        - name: add a hybrid connection to the app
+          text: >
+            az functionapp hybrid-connection add -g [myResourceGroup] -n [myAppName] -s [slotName]
+            --namespace [hybridConnectionNamespaceName] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['functionapp hybrid-connection remove'] = """
+    type: command
+    short-summary: disconnects the specified hybrid connection from the specified functionapp
+    examples:
+        - name: remove the hybrid connection from the functionapp
+          text: >
+            az functionapp hybrid-connection remove -g [myResourceGroup] -n [myAppName] -s [slotName]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['webapp hybrid-connection'] = """
+    type: group
+    short-summary: commands that list, add, or remove hybrid connections from a sepcific app
+"""
+
+helps['webapp hybrid-connection list'] = """
+    type: command
+    short-summary: lists all the hybrid connections set on the specified app
+    examples:
+        - name: list the hybrid connections
+          text: >
+              az webapp hybrid-connection list -g [myResourceGroup} -n {myAppName] -s [slotName]
+"""
+
+helps['webapp hybrid-connection add'] = """
+    type: command
+    short-summary: adds an existing hybrid connection to the specified app
+    examples:
+        - name: add a hybrid connection to the app
+          text: >
+            az webapp hybrid-connection add -g [myResourceGroup] -n [myAppName] -s [slotName]
+            --namespace [hybridConnectionNamespaceName] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['webapp hybrid-connection remove'] = """
+    type: command
+    short-summary: removes an existing hybrid connection from the specified app
+    examples:
+        - name: remove the hybrid connection from the app
+          text: >
+            az webapp hybrid-connection remove -g [myResourceGroup] -n [myAppName] -s [slotName]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
```

Comment:
Do we need a `show` command, or because hybrid-connection is not any kind of resource?